agent/logger/log.go

```diff
@@ -14,87 +14,162 @@
 package logger
 
 import (
+	"fmt"
```

Comment:
Any test we can add to double check the log is generated as we expected or maybe unit test to make sure the functions in the file work as expected?

Reply:
I can try to write some unit tests for the seelog formatter funcs

Reply:
done